src/inmanta/server/services/orchestrationservice.py

```diff
@@ -1192,16 +1197,22 @@ async def release_version(
                 await model.mark_done(connection=connection)
                 return 200, {"model": model}
 
-            if push:
+            is_using_new_scheduler = opt.server_use_resource_scheduler.get()
+            if push or is_using_new_scheduler:
```

Comment:
I would not mix these two code path, make them clear, separate branches in the control flow, so we have no issues when deleting the old code path